### PR TITLE
Planning Center Provider

### PR DESCRIPTION
**Pull request by duncanmcclean**
This pull request implements a Socialite provider for [Planning Center](https://www.planningcenter.com/).

## Reference

* [Planning Center's oAuth Docs](https://developer.planning.center/docs/#/overview/authentication)

### CODE REVIEW

**Review by lucasmichot — APPROVED** [review]
Thx for the PR @duncanmcclean 
GTM @atymic

File: src/PlanningCenter/Provider.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+use GuzzleHttp\RequestOptions;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    public const IDENTIFIER = 'PLANNINGCENTER';
```
Comment (lucasmichot) [inline src/PlanningCenter/Provider.php:14]: Should this not be, `PLANNING_CENTER` ?

File: src/PlanningCenter/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Installation & Basic Usage
+
+Please see the [Base Installation Guide](https://socialiteproviders.com/usage/), then follow the provider specific instructions below.
+
+### Add configuration to `config/services.php`
+
+```php
+'planningcenter' => [
+  'client_id' => env('PLANNING_CENTER_CLIENT_ID'),
```
Comment (lucasmichot) [inline src/PlanningCenter/README.md:15]: You might need to update the provider identifier to `PLANNING_CENTER`

**Review by atymic — APPROVED** [review]

**Comment by atymic** [comment]
https://github.com/SocialiteProviders/PlanningCenter/releases/tag/5.0.0

**Comment by atymic** [comment]
Please PR to website :)

**Pull request merged**